apps/trading-e2e/src/support/step_definitions/trading-page.step.ts

```diff
@@ -55,14 +55,14 @@ Given('I am on the trading page for an active market', () => {
   mockMarket(MarketState.Active);
 
   cy.visit('/markets/market-id');
-  cy.wait('@Market');
+  cy.wait('@Market', { timeout: 3000 });
```

Comment:
question (non-blocking): Why are these needed now? Seemed to be working nicely without it before?

Reply:
I think I spotted it timing out a few times so I increased it by a bit. But looking now at the test run history I can't even find it anymore so probably not needed. Removed.